extensions/narayana-jta/runtime/src/main/java/io/quarkus/narayana/jta/runtime/context/TransactionContext.java

```diff
@@ -212,5 +222,13 @@ void destroy() {
                     .collect(Collectors.toMap(ContextInstanceHandle::getBean, ContextInstanceHandle::get));
         }
 
+        @Override
+        public void beforeCompletion() {
+        }
+
+        @Override
+        public void afterCompletion(int i) {
```

Comment:
Despite the argument is not used it could be named better.

```suggestion
        public void afterCompletion(int status) {
```

Reply:
Got it. I'll incorporate the suggestion.